src/app/pages/home/profile/profile.page.html

```diff
@@ -1,38 +1,25 @@
-<ion-content>
-
-  <ion-list>
-
-    <ion-card button style="height:auto;width:auto;" [routerLink]="['/card', card.id]" *ngFor="let card of cards" lines="none">
-        <img [src]="card.frontImg" alt=""/>
-    </ion-card>
-
-    <div class="floating-fab-btn">
-      <ion-fab vertical="top" horizontal="start" edge slot="fixed" padding>
-        <ion-fab-button size="small" (click)="createCard()">
-          <ion-icon name="add" size="small"></ion-icon>
-        </ion-fab-button>
-      </ion-fab>
-    </div>
-      <ion-card padding>
-          <ion-avatar slot="start">
-            <img src="./assets/imgs/avatar.svg">
-          </ion-avatar>
-        <div style="position:relative;right:20px;bottom:43px;height:1px;">
-          <hr width="25%">
-          <hr width="35%">
-          <hr width="30%">
-        </div>
-        <ion-label>
-          <hr width="55%">
-          <hr width="50%">
-          <hr width="25%">
-          <hr width="15">
-        </ion-label>
-        <ion-icon name="square-outline" size="large"></ion-icon><ion-icon name="square-outline" size="large"></ion-icon><ion-icon name="square-outline" size="large"></ion-icon>
-      </ion-card>
-  </ion-list>
-
-
+<ion-toolbar>
+  <ion-title slot="start">Cards</ion-title>
+  <div class="floating-fab-btn">
+    <ion-fab vertical="top" horizontal="end">
+      <ion-fab-button size="small" (click)="createCard()">
```

Comment:
While I agree with the need for this change, I don't really see how removing the custom card icon is related to this issue.

Reply:
It's a part of updating the view for the profile page. The card wallet architecture doesn't call for a blank card template as you've said before, so I removed it to produce the updated view.